docs/guide/static-deploy.md

```diff
@@ -0,0 +1,265 @@
+# Deploying a Static Site
+
+The following guides are based on some shared assumptions:
+
+- You are using the default build output location (`dist`). This location [can be changed using `build.outDir`](https://vitejs.dev/config/#build-outdir), and you can extrapolate instructions from these guides in that case.
+- Vite is installed as a local dev dependency in your project, and you have setup the following npm scripts:
+
+```json
+{
+  "scripts": {
+    "build": "vite build",
+    "serve": "vite serve"
+  }
+}
+```
+
+::: tip NOTE
+These guides provide instructions for performing a static deployment of your Vite site. Vite also has experimental support for Server Side Rendering. SSR refers to front-end frameworks that support running the same application in Node.js, pre-rendering it to HTML, and finally hydrating it on the client. Check out the [SSR Guide](./ssr) to learn about this feature. On the other hand, if you are looking for integration with traditional server-side frameworks, check out the [Backend Integration guide](./backend-integration) instead.
+:::
+
+## Building The App
+
+You may run `yarn build` command to build the app.
+
+```bash
+$ yarn build
+```
+
+By default, the build output will be placed at `dist`. You may deploy this `dist` folder to any of your preferred platforms.
+
+### Testing The App Locally
+
+Once you've built the app, you may test it locally by running `yarn serve` command.
+
+```bash
+$ yarn build
```

Comment:
I think we should use `npm` by default and add a note "use equivalent commands to run the scripts if you are using Yarn or other package managers"